pkg/web/routes.go

```diff
@@ -125,6 +125,9 @@ func (s *Server) setupRoutes() (err error) {
 		v1.POST("/authenticate", s.Authenticate)
 		v1.POST("/reauthenticate", s.Reauthenticate)
 
+		// User Profile Management
+		v1.POST("/change-password", authenticate, s.ChangePassword)
```

Comment:
This route allows the logged in user to change their password -- I'm thinking it is part of a suite of user profile tools?

Reply:
This would be used on a user profile page (that's specific to the logged in user) while the API route below would be used on a users management page that would make the option to change passwords (and other settings in the future) only available to the admin?

Reply:
That's correct!